packages/odyssey-react/src/components/Link/Link.module.scss

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.link {
+  color: cv('blue', '500');
+  text-decoration: none;
+
+  &:hover {
+    color: cv('blue', '500');
+    text-decoration: underline;
+  }
+
+  &:visited {
+    color: cv('blue', '500');
+  }
+
+  /* stylelint-disable selector-no-qualifying-type */
+  &[target='_blank']:not([href^='mailto:'])::after {
```

Comment:
can we revisit this code here? I would love to see a separate class for this style, `linkExternal`. And have the application of this class entirely controlled through a prop for `Link` `external`

Reply:
Fixed, however, it is using the `target` prop to apply the `linkExternal` styling, if we want to change this I would suggest we do it as part of the "react next steps" epic.